include/cppoptlib/solver/lbfgsb.h

```diff
@@ -302,6 +302,8 @@ class Lbfgsb : public Solver<function_t> {
 
  private:
   int dim_;
+  vector_t lower_bound_;
```

Comment:
We had a default value before (-inf, info). We should initialize these bound again by default if no other bound is given

Reply:
Do you mean set a default in `InitializeSolver`? For functions where `Dim = Eigen::Dynamic`, I'll need `dim_` to be initialized to set default bounds of the correct size.

The way I have it now, `Lbfgsb` has to be initialized with both bounds. Why would someone use a bounded solver if they don't want to impose any bounds?